### PR TITLE
262828 Fix up File Upload/Download feature (WebUI)

### PR DESCRIPTION
**Pull request by andygibbonsdefra**

### CODE REVIEW

File: EST.MIT.Web/Services/UploadAPI.cs [hunk 1]
```diff
@@ -85,4 +85,41 @@ public async Task<byte[]> GetFileByFileNameAsync(string fileName)
         }
     }
 
+
```
Comment (daveonplanetearth) [inline EST.MIT.Web/Services/UploadAPI.cs:88]: Do we still need the code here?

File: EST.MIT.Web.Test/Repo/UploadRepositoryTests.cs [hunk 2]
```diff
@@ -59,57 +59,57 @@ public async Task HandleHttpResponseError_Handed_FailCode()
...
-
-        var repo = new UploadRepository(factory);
-
-        var response = await repo.GetFileByFileNameAsync(fileName);
-
-        response.StatusCode.Should().Be(HttpStatusCode.NotFound);
-        var content = await response.Content.ReadAsStringAsync();
-        content.Should().BeEmpty();
-    }
+    //[Fact]
```
Comment (daveonplanetearth) [inline EST.MIT.Web.Test/Repo/UploadRepositoryTests.cs:62]: Do we still need these tests?

File: EST.MIT.Web/Services/UploadAPI.cs (hunk 1 above)
Comment (andygibbonsdefra) [reply EST.MIT.Web/Services/UploadAPI.cs]: Removed

File: EST.MIT.Web.Test/Repo/UploadRepositoryTests.cs (hunk 2 above)
Comment (andygibbonsdefra) [reply EST.MIT.Web.Test/Repo/UploadRepositoryTests.cs:62]: Removed

Comment (andygibbonsdefra) [reply EST.MIT.Web.Test/Repo/UploadRepositoryTests.cs:62]: Actually, fixing tests.... hang on...

Comment (andygibbonsdefra) [reply EST.MIT.Web.Test/Repo/UploadRepositoryTests.cs]: Re-instated and fixed tests

**Review by daveonplanetearth — APPROVED** [review]

**Pull request merged**